ironfish-rust-nodejs/src/xchacha20poly1305.rs

```diff
@@ -2,12 +2,144 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-use ironfish::xchacha20poly1305::{self, EncryptOutput};
+use ironfish::xchacha20poly1305::{
+    self, EncryptOutput, XChaCha20Poly1305Key, KEY_LENGTH as KEY_SIZE, SALT_LENGTH as SALT_SIZE,
+    XNONCE_LENGTH as XNONCE_SIZE,
+};
 use napi::{bindgen_prelude::*, JsBuffer};
 use napi_derive::napi;
 
 use crate::to_napi_err;
 
+#[napi]
```

Comment:
Do you think it'd be helpful / less confusing if we namespace these exports? Similar to how we do with multisig - definitely something I didn't know about before 
https://github.com/iron-fish/ironfish/blob/staging/ironfish-rust-nodejs/src/multisig.rs#L60
https://github.com/iron-fish/ironfish/blob/staging/ironfish-rust-nodejs/index.d.ts#L279

Reply:
Yeah, I can update it